.github/workflows/image-push-release.yml

```
@@ -1,4 +1,8 @@
 name: "Push images on release"
+
+env:
+  BUILD_PLATFORM: linux/amd64,linux/arm64,linux/ppc64le
```

```suggestion
  BUILD_PLATFORM: [linux/amd64,linux/arm64,linux/ppc64le]
```
should it be this for consistency with the other files?

---

In fact, the brackets are not needed here. Do you have a different opinion?

---

No, it was a nit for the look of it :) 
no other reason.

---

I'm not sure if the github action works if the brackets are added 🤔

---

https://github.com/metallb/metallb/blob/e4c0a7b1b01d4ac978deaa2af4227403b43d759f/.github/workflows/publish.yaml#L104
The value of platforms should be a comma-separated string, not a list

---

Okay, good to leave as is!